hw5-10_to_do_list/react_ts/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "hw5-10_to_do_list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "app": "node_modules/.bin/webpack serve & json-server --watch database/db.json",
+    "build": "node_modules/.bin/webpack",
```

```suggestion
    "build": "webpack",
```
node_modules/.bin - it is a default folder to search binaries )